Added pricing-and-billing.md and reworked faq.md (#690)
fermyon/developer: content/cloud/pricing-and-billing.md

title = "Pricing Plans and Billing"
template = "cloud_main"
date = "2023-06-20T00:07:00Z"
enable_shortcodes = true
[extra]
url = "https://github.com/fermyon/developer/blob/main//content/cloud/payment-and=billing.md"

---
- [Fermyon Cloud Pricing](#fermyon-cloud-pricing)
- [General Information](#general-information)
  - [Payment](#payment)
  - [Refund](#refund)
  - [Support](#support)
  - [Quota](#quota) 
- [Next Steps](#next-steps)

## Fermyon Cloud Pricing
Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quota](#quota).

As described in detail on the [Fermyon Pricing page](fermyon.com/pricing), there are 3 different plans crafted to fit your workloads' requirements: Starter, Growth, and Enterprise. 

> The Starter Plan provides a broad range of Fermyon Cloud services for you to try out, free of charger, forever. If you find that you're hitting the upper limits of the Starer Plan, we recommend promoting your account to the Growth Plan, where you can enjoy the same services with a higher quota limit. Lastly, if the serives offered don't quite meet your needs in the Growth Plan, [reach out](mailto:sales@fermyon.com) to the Fermyon team and we'll work to create a customized Enterprise Plan that will best meet your requirements. 

Reviewer: ```suggestion
The Starter Plan provides a broad range of Fermyon Cloud services for you to try out, free of charge, forever. If you find that you're hitting the upper limits of the Starter Plan, we recommend promoting your account to the Growth Plan, where you can enjoy the same services with a higher quota limit. Lastly, if the services offered don't quite meet your needs in the Growth Plan, [reach out](mailto:sales@fermyon.com) to the Fermyon team and we'll work to create a customized Enterprise Plan that will best meet your requirements. 
```

> > Please note that Growth and Enterprise Plans are billed on a monthly reoccuring basis. Start Plan is a free subscription, without an expiration date. 

Reviewer: ```suggestion
> Please note that Growth and Enterprise Plans are billed on a monthly reoccuring basis. The Starter Plan is a free subscription, without an expiration date. 
```

> > Please note that Growth and Enterprise Plans are billed on a monthly reoccuring basis. Start Plan is a free subscription, without an expiration date. 

Reviewer 2: ```suggestion
> Please note that Growth and Enterprise Plans are billed on a monthly reoccurring basis. Start Plan is a free subscription, without an expiration date. 
```

(as well as Fisher's spelling spots)

## General Information

### Payment

**I just signed up for the Growth Plan, when should I expect my first charge?**

> You should receive your first charge immediately after signing up for the Growth Plan. You can expect your second charge on the monthly anniversary of your signup.

Reviewer: ```suggestion
You should receive your first charge shortly after signing up for the Growth Plan. You can expect your second charge on the monthly anniversary of your signup.
```

**What payment methods for you accept?**

> Most. One of the main reasons we chose to work with industry leader Stripe is because of the huge scope of payment options they offer, starting with the normal credit / debit card options, but extending to ACH, SEPA, Apple Pay / Google Pay and many more including a larger variety of global options. For more details and a list to all Stripe payment options see this Stripe Resource. 

Reviewer: > see this Stripe Resource

Missing link?

> Most. One of the main reasons we chose to work with industry leader Stripe is because of the huge scope of payment options they offer, starting with the normal credit / debit card options, but extending to ACH, SEPA, Apple Pay / Google Pay and many more including a larger variety of global options. For more details and a list to all Stripe payment options see this Stripe Resource. 

Reviewer 2: Spare me the Stripe sales pitch #badattitude - list the important ones (maybe bulleted), link for other ones, and don't clutter it telling me how awesome Stripe is.

```suggestion
We accept all forms of payment supported by our payment partner Stripe. These include:

* Credit and debit cards
* ACH
* SEPA
* Apple Pay / Google Pay

For a full list to accepted payment methods see ***STRIPE LINK*** 
```

**Can I pay every month manually?**

> No, at this time Fermyon Cloud only accepts reoccuring payments. If at any point you decide you no longer want to continue with your paid plan, you are eligable to cancel your subscription. 

Reviewer: ```suggestion
No, at this time Fermyon Cloud only accepts reoccurring payments. If at any point you decide you no longer want to continue with your paid plan, you are eligible to cancel your subscription. 
```

> **What happens if there's an issue with my payment method during a reoccuring charge?**

Reviewer: ```suggestion
**What happens if there's an issue with my payment method during a reoccurring charge?**
```

> **What happens if there's an issue with my payment method during a reoccuring charge?**

Reviewer 2: ```suggestion
**What happens if there's an issue with my payment during a reoccurring charge?**
```

> You will receive an email from Fermyon informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Subscription.

Reviewer: ```suggestion
You will receive an email from Fermyon informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Plan.
```

> You will receive an email from Fermyon informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Subscription.

Reviewer 2: ```suggestion
Fermyon will send you an email informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Subscription.
```

(suggesting the change because we're not in control of them receiving it)

Also, do we only send one email?

Not quite sure about the tone of "that the payment method is invalid."  Okay, something went wrong in the payment.  That doesn't mean the method is "invalid."

Reviewer: We (or rather Stripe) send a few e-emails. We continue to retry for a number of days, after which they are then downgraded to the Starter plan.

Stripe uses the term "declined" to cover payments that the issuer declines. So perhaps this?

"Fermyon will send you an email informing you that a recent payment has been declined and..."

**How do I change my credit card used for billing by Fermyon Cloud?**

> This can be done online on the [Fermyon Cloud Dashboard](cloud.fermyon.com) under _Account Settings --> Billing_. 

Reviewer: Missing newline character above this line

> This can be done online on the [Fermyon Cloud Dashboard](cloud.fermyon.com) under _Account Settings --> Billing_. 

Reviewer 2: Is there no direct link to the billing page?

Author: Hmm, I think the URL varies by user but I could be wrong. @bacongobbler - do you know if there's a direct URL to the billing tab?

> 

Reviewer: delete this double-newline character

**How long is the Starter Plan available? Is there a time limit after which the free tier is unavailable to me?**

> At this time the Starter Plan has no time limit. As long as your resource utilization remains below the monthly limits you can use Fermyon Cloud, for free, as long as you want it to. We reserve the right to change that policy, but for now we’re thrilled to be able to provide this service to you completely free.

Reviewer: ```suggestion
At this time the Starter Plan has no time limit. As long as your resource utilization remains below the monthly limits, you can use Fermyon Cloud, for free, as long as you want it to. We reserve the right to change that policy, but for now we’re thrilled to be able to provide this service to you completely free.
```

### Invoice 

**Where can I see my bill?**

> You can view all past invoices in the customer portal provided by Stripe. Please navigate to [Fermyon Cloud Dashboard](https://cloud.fermyon.com) _—> Account settings —> Billing_.

Reviewer 2: Direct link?

**Can I see multiple past bills?**

Yes. You can view all past bills/invoices for your account in 
[Fermyon Cloud Dashboard](https://cloud.fermyon.com) under _Account settings —> Billing_.

### Refund

**How do I dispute a charge from Fermyon Cloud?**

> We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com)

Reviewer: ```suggestion
We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com).
```

### Support

**How do I report a problem with Fermyon Cloud?**
We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com). For our Starter users, we recommend reaching out to our [Discord community](https://discord.gg/AAFNfS7NGf) for assistance. 

> **What support is offered with Gthe Growth Plan?**

Reviewer: ```suggestion
**What support is offered with the Growth Plan?**
```

> We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.

Reviewer 2: "Monday through Friday" US time or...?

**Can I purchase a better level of support?**

Call us and we’ll work to understand the situation and see if we can create a solution to meet your needs.

> 

Reviewer: delete this newline

> 

Reviewer: delete this newline

**Does Fermyon Cloud Starter Plan or Growth Plan come with any Service Level Agreements and if so, what are they?**

Fermyon Cloud is an Open Beta service, which currently does not guarentee any Service Level Agreement. 

**Does my use of Fermyon Cloud place any restrictions on the currently open-source Spin?**

No. Spin remains 100% open source and we remain highly committed to both the Spin open source community and open source in general. You can read more about our Open Source Promise [here](https://www.fermyon.com/open-source-promise).

> **Do you use data subprocessors? If so, where can I see a list of those you rely on.**

Reviewer 2: ```suggestion
**Do you use data subprocessors? If so, where can I see a list of those you rely on?**
```

> For an up-to-date list of Fermyon’s data subprocessors please visit https://www.fermyon.com/sub-processors.

Reviewer: ```suggestion
For an up-to-date list of Fermyon’s data subprocessors please visit <https://www.fermyon.com/sub-processors>.
```

This ensures the URL is converted into a hyperlink.

https://daringfireball.net/projects/markdown/syntax#autolink

**Can I purchase a Service Level Agreement?**

> As part of Fermyon Cloud's Starter Plan you have access to our vibrant discord community and as part of the Growth Plan you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com)

Reviewer: ```suggestion
As part of Fermyon Cloud's Starter Plan, you have access to our vibrant [Discord community](https://discord.gg/AAFNfS7NGf). As part of the Growth Plan, you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com)
```

> As part of Fermyon Cloud's Starter Plan you have access to our vibrant discord community and as part of the Growth Plan you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com)

Reviewer 2: ```suggestion
As part of Fermyon Cloud's Starter Plan you have access to our vibrant discord community and as part of the Growth Plan you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com).
```

(as well as Fisher's changes of course)

> If these do not meet your needs, call us at xxx.xxx.xxxx and we can discuss your situation in greater detail to see if we can develop a custom solution that fits.

Reviewer: missing phone number?

> If these do not meet your needs, call us at xxx.xxx.xxxx and we can discuss your situation in greater detail to see if we can develop a custom solution that fits.

Reviewer 2: Phone call in the first instance?  Rather than sales email?  Should we mention this for the Enterprise Plan too then?

### Quota

**How can I tell if I’m approaching my plan’s current quota limits?**

> Fermyon will notify you via email when you reach 75%, 90%, and 100% usage for your limits. In the future we will be adding in Fermyon Cloud UI notifications as well and the ability to configure alerts.

Reviewer 2: Seems like "What happens when I reach my quota limits?" would be a natural follow-on that I don't think is covered...?

Author: Good point! I added a QA pair to address this gap

**How can I increase my limits above the limits set by the Growth Plan?**

Please reach out to [our team](mailto:sales@fermyon.com) and we'll create a customized solution to meet your current needs. 

> **I'm interested in increase a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**

Reviewer: ```suggestion
**I'm interested in increasing a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**
```

> **I'm interested in increase a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**

Reviewer 2: ```suggestion
**I'm interested in increasing a quota limit on a resource not listed in Fermyon's pricing plan.  Can you help?**
```

> Fermyon will happily consider the request! Please reach out to [our team](mailto:sales@fermyon.com) sharing the desired resource and quota limit, and we'll work to assist you. You can see a list of Fermyon Cloud quopta limits [here](https://developer.fermyon.com/cloud/faq#quota-limits).

Reviewer: ```suggestion
Fermyon will happily consider the request! Please reach out to [our sales team](mailto:sales@fermyon.com) sharing the desired resource and quota limit, and we'll work to assist you. You can see a list of Fermyon Cloud quota limits [here](https://developer.fermyon.com/cloud/faq#quota-limits).
```

title = "Pricing Plans and Billing"
template = "cloud_main"
date = "2023-06-20T00:07:00Z"
enable_shortcodes = true
[extra]
url = "https://github.com/fermyon/developer/blob/main//content/cloud/payment-and=billing.md"

---
- [Fermyon Cloud Pricing](#fermyon-cloud-pricing)
- [General Information](#general-information)
  - [Payment](#payment)
  - [Refund](#refund)
  - [Support](#support)
  - [Quota](#quota) 
- [Next Steps](#next-steps)

## Fermyon Cloud Pricing

> Fermyon Cloud offers subscription plans to help you accomplish your workload goals which require an increase Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quotas](#quota).

Reviewer 3: ```suggestion
Fermyon Cloud offers subscription plans to help you accomplish your workload goals which require an increased Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quotas](#quota).
```

- [Next Steps](#next-steps)

## Fermyon Cloud Pricing

> Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quotas](#quota).

Reviewer 4: I'm not sure I'm seeing any change to this?

> Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quotas](#quota).

Author: I think it's because the comment preview didn't refresh. Here is the latest verbiage:

> Fermyon Cloud offers subscription plans to help you accomplish your workload goals which require an increase Fermyon Cloud usage. 

If that sounds good to you, I'll resolve this comment!

**How do I report a problem with Fermyon Cloud?**
We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com). For our Starter users, we recommend reaching out to our [Discord community](https://discord.gg/AAFNfS7NGf) for assistance. 

**What support is offered with the Growth Plan?**

> We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday from 8am to 6pm Eastern Time Zone. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.

Reviewer 4: Maybe add the Time zone abbreviation to make sure there's no confusion (ET): https://www.timeanddate.com/time/zones/et

> We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday from 8am to 6pm Eastern Time Zone. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.

Reviewer 4: "In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers."

I think this is telling you to not use the support included in the Growth plan, if we "promise" answers to be provided in less than one business day. Maybe just delete this paragraph.

For an up-to-date list of Fermyon’s data subprocessors please visit https://www.fermyon.com/sub-processors.

**Can I purchase a Support Service Level Agreement?**

> As part of Fermyon Cloud's Starter Plan you have access to our vibrant discord community and as part of the Growth Plan you have access to next business day support through [support@fermyon.com](mailto:support@fermyon.com)

Reviewer 4: Why "vibrant"? It sounds fun, but maybe I'm not looking for fun if I need support :-)

Please reach out to [our team](mailto:sales@fermyon.com) and we'll create a customized solution to meet your current needs. 

> **I'm interested in increasing a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**

Reviewer 4: Is this any different than the above question? I can see the nuance on the technical implementation, but just having one Q about raising quota limits, seems to be sufficient.

Author: I reworked the first question to make the distinction more clear. The first question is around promoting from Growth Plan to Enterprise Plan. The second question is about increasing quota limits not explicitly included in the pricing model - this could mean the user is a good fit for Enterprise, but it could also mean they have requirements we aren't able to meet quite yet as a Cloud (due to them requesting an increase in a resource we can't scale et), or that our pricing model should include that resource going forward.

Does that help separate them a bit more, or should we consider consolidating the question?
